### DISCUSSION TITLE
[38]  Generic temperature sensor 1

### DISCUSSION THREAD

**Discussion by giuian69** [merbanan/rtl_433#2615] [Q&A]
Hi, i have and old outdoor temperature sensor T-Logic.
Currently i run rtl_433 on an esp32 with OpenmqttGateway installed.
The MQTT message that i receive is:
{
    "model": "Generic-Temperature",
    "id": 65,
    "battery_ok": 0,
    "temperature_C": -135.2,
    "protocol": "Generic temperature sensor 1",
    "rssi": -29,
    "duration": 1037992
}

But the temperature on display is 29.8 °C and depending on temperature the "id" change from 64 to 65 and maybe other values when the temperature change more.
How i can read the raw value for try to decode correctly this sensor?
Someone have encountered the same issue?
i know that the easy way is to buy another well supported sensor, but if i can try somewhat, i will try.
Sorry for my English
Thanks

**Comment by zuckschwerdt** [comment]
The timings might be similar with our Generic-Temperature (PPM 2000/4000). But the data seems very different. We'd need a sample file (https://triq.org/rtl_433/ANALYZE.html) to check the timings and then later a number of data codes to find the fields. But that will need an RTL-SDR.

**Reply by giuian69** [reply]
Ok thanks. I will try with a PI and a RTL-SDR
